.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/pointer-constraints-demo-client"]
+	path = examples/pointer-constraints-demo-client
+	url = https://github.com/Laaas/hello_wayland.git
```

Comment:
Haha, this isn't going to cut it. Look at some of the other example clients, like idle or layer-shell.